attributes/default.rb

```diff
@@ -20,3 +20,15 @@
 
 default['selinux']['status'] = 'enforcing'
 default['selinux']['booleans'] = {}
+default['selinux']['packages'] = case node['platform_family']
```

Comment:
This should move to the install resource as a property with defaults per platform